### ISSUE TITLE
Inital/Investigation upgrade to Debian Jessie (8)

### ISSUE THREAD

**Issue by grndlvl** [grndlvl/dstack#78]
Branch: feature/MoveToDebian
- [x] 1. Update to using Chef Zero
- [x] 2. Use a provisionless bento box
- [x] 3. Update cookbooks as necessary
- [x] 4. Create follow-up tasks
- [x] 5. Create follow-up ticket Debug drupal::varnish cookbook
- [x] 6. Remove config.yml used for testing
- [x] 7. Create another milestone specifically for Debian Jessie? Is there a sub milestone or subtasks?
- [x] 9. Test default-web cookbook

**Comment by grndlvl** [comment]
Do not merge until all follow-up tickets have been addressed.

### Follow-up tickets
- #79
- #80
- #81
- #82
- #83
- #84
- #85
- #86
- #87
- #88
- #92
